runtime/bindings/python/iree/runtime/benchmark.py

```diff
@@ -97,6 +97,9 @@ def benchmark_module(module, entry_functiong=None, inputs=[], timeout=None, **kw
         args.append(f"--input={shape}x{abitype}={values}")
     args.append(f"--module=-")
 
+    # DEBUG - DO NOT SUBMIT
+    print(f"running process with args: '{args}'")
+
     try:
         benchmark_process = subprocess.run(
```

Comment:
https://github.com/openxla/iree/actions/runs/6003448260/job/16282146758?pr=14850#step:6:2904
```
FileNotFoundError: [Errno 2] No such file or directory: '/work/full-build-dir/runtime/bindings/python/iree/runtime/iree-benchmark-module'
```

Soooo... did this ever work?

Reply:
@KoolJBlack / @rsuderman I'm confused by this script. It appears to be broken and the tests have never been enabled. Is it being used anywhere?